bindings/matrix-sdk-ffi/src/notification_settings.rs

```diff
@@ -208,15 +208,35 @@ impl NotificationSettings {
             .spawn(async move {
                 let notification_settings = self.sdk_notification_settings.read().await;
                 let mode = notification_settings
-                    .get_default_room_notification_mode(is_encrypted, active_members_count)
+                    .get_default_room_notification_mode(is_encrypted.into(), active_members_count)
                     .await;
                 mode.into()
             })
             .await
             .unwrap()
     }
 
-    /// Restores the default notification mode for a room
+    /// Set the default room notification mode
+    ///
+    /// # Arguments
+    ///
+    /// * `encrypted` - whether the mode is for encrypted rooms
+    /// * `one_to_one` - whether the mode is for direct chats
```

Comment:
Can we precise "direct chats involving two people"?

Reply:
👍